weis/test/run_examples.py

```diff
@@ -2,23 +2,29 @@
 from weis.test.utils import execute_script

```

Comment:
I think this file was removed in the Fri hackathon and everything should now be in `test_examples.py`.  You might want to `git merge develop` into your fork to identify conflicts?